tests/system-tests/rdscore/internal/rdscorecommon/egress-ip.go

```diff
@@ -496,7 +499,7 @@ func gracefulNodeReboot(nodeName string) error {
 		return fmt.Errorf("failed to retrieve node %s object due to: %w", nodeName, err)
 	}
 
-	glog.V(rdscoreparams.RDSCoreLogLevel).Infof("Cordoning node %q", nodeName)
+	glog.V(100).Infof("Cordoning node %q", nodeName)
```

Comment:
any particular reason to switch from `rdscoreparams.RDSCoreLogLevel` variable?

Reply:
yes, I always use the general default value (100) for sub-functions. it will be easy to handle it if will be decided to move some methods to the upper level